### DISCUSSION TITLE
Support for On-Demand Materialized Views

### DISCUSSION THREAD

**Discussion by Mosquid** [typegoose/typegoose#785] [Q&A]
Hi. I wanted to check whether there's a way to create a "materialized view" using Typegoose?

https://www.mongodb.com/docs/manual/core/materialized-views/#std-label-manual-materialized-views

**Comment by smolinari** [comment]
Since Typegoose is only a wrapper of Mongoose, the right question would be, does Mongoose support it? It doesn't. And then, if Mongoose doesn't support it, the next place to look is the MongoDB Node driver, if it supports it. And the way it looks, it doesn't. 

At least from my short research, I couldn't see it supports it.

Scott

**Reply by Mosquid** [reply]
It seems like mongoose does support it
https://mongoosejs.com/docs/models.html#views

**Reply by smolinari** [reply]
Um, those are regular views. You asked about materialized views. :grin: 

Scott
